sycl/test/plugins/lspi_gpu_cuda.cpp

```diff
@@ -0,0 +1,24 @@
+// REQUIRES: gpu, cuda
+
+// RUN: lspi --verbose >%t.default.out
```

Comment:
```suggestion
// RUN: sycl-ls --verbose >%t.default.out
```
I encourage to run `check-*` locally before pushing to remote to avoid bugs like this.

Reply:
OK